apps/aechannel/src/aesc_channels.erl

```diff
@@ -28,8 +28,10 @@
          participant/1,
          total_amount/1,
          initiator_amount/1,
+         participant_amount/1,
          channel_reserve/1,
-         sequence_number/1]).
+         sequence_number/1,
```

Comment:
Are we going with `sequence_number`?
I've just been changing it to `round` in my branch...